supplier_api_tester/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.4.2
+- Updating availability tests
```

Comment:
Removing tests for the list of available dates.

Reply:
thanks